doc-content/enterprise-only/project-data/preview-editor-create-test-scenario-mortgages-example-proc.adoc

```diff
@@ -0,0 +1,76 @@
+[id='preview-editor-create-test-scenario-mortgages-example-proc']
+= Creating test scenario using the sample Mortgages project
+
+In this chapter, as an example for better understanding, you will create and execute a test scenario from the sample *Mortgages* project shipped with {CENTRAL} using the *Test Scenario (Preview)* editor. The test scenario that you will create is based on the *Pricing loans* guided decision table from the *Mortgages* project.
+
+.Prerequisite
+Ensure that the Test Scenario (Preview) editor is enabled.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click *Mortgages*.
+. If the project is not listed under *Projects*, from *MySpace*, click the three dots (image:cases/dotdotdotbutton.png[]) in the upper-right corner of the page.
+. Click *Try Samples* -> *Mortgages* -> *OK*. The Assets window appears.
```

Comment:
The statement "The Assets window appears." is the result of the action and hence should come in the next line.

Reply:
Changed as suggested.